contrib/tkey-qemu.dockerfile

```
@@ -1,40 +1,41 @@
+# Copyright (C) 2023 Tillitis AB
+# SPDX-License-Identifier: GPL-2.0-only
 
 # This is the tag in the https://github.com/tillitis/tillitis-key1 repo with
 # the firmware that our TKey/QEMU will run. The published tkey-qemu image will
 # have this as part of its name, the tag is then used for versioning the image
 # (could be updates to the TKey QEMU machine).
-ARG TKEYREPO_TAG=TK1-23.03.1
+ARG TKEYREPO_TAG=TK1-23.03.2
```

Perhaps this could be the current release version?

```suggestion
ARG TKEYREPO_TAG=TK1-24.03
```